CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+This doc describes all notable changes to Diem Core and Diem Framework between releases.
+
+
+## [diem-core-v1.1.5] - 2021-03-02
+*[Added]*
+
+* Add NetworkIdentity to metrics dashboard (#7669)
+* Add health check of node status via json-rpc (#7498)
+* Add genesis smoke test to backup & restore tooling (#7465)
+
+*[Changed]*
+
+* Rename mint to faucet container for readability in testnet. (#7618)
+* Use branch specific CI container (see #7649)
+* Backup & restore tooling to accept genesis or latest waypoint (#7465)
+
+*[Security]*
+
+* Add RUSTSEC-2021-0020 to audit ignore list (#7723)
```

Comment:
Can you elaborate please? From what I understand, the RUSTSEC-2021-0020 is a vulnerability and we choose to ignore it. Why don't we upgrade to a patched version as define here https://rustsec.org/advisories/RUSTSEC-2021-0020.html? Sorry if my question is stupid.

Reply:
@frebours Young put more details in the PR which made this change https://github.com/diem/diem/pull/7723